.gitignore

```diff
@@ -51,36 +51,9 @@ coverage.xml
 .hypothesis/
 .pytest_cache/
 
-# Translations
```

Comment:
This just general cleanup or somehow related to the work of this PR specifically?

Reply:
just general cleanup